project_test/translate_rapid_pro_keys.py

```diff
@@ -0,0 +1,54 @@
+import time
+from os import path
+
+from core_data_modules.traced_data import Metadata
+from core_data_modules.traced_data.io import TracedDataCoda2IO
+from dateutil.parser import isoparse
+
+from project_test.lib.test_schemes import CodeSchemes
+
+class TranslateRapidProKeys(object):
+    RAPID_PRO_KEY_MAP = [
+        # List of (new_key, old_key)
+        ("uid", "avf_phone_id"),
+
+        ("internet_working_raw", "Response_2 (Value) - internet_working_poll" ),
+        ("internet_working_run_id", "Response_2 (Run ID) - internet_working_poll"),
+        ("sent_on", "Response_2 (Time) - internet_working_poll"),
+
+        ("water_filter_raw", "Filter_Working (Value) - water_filter_poll"),
+        ("water_filter_run_id", "Filter_Working (Run ID) - water_filter_poll"),
+        ("sent_on", "Filter_Working (Time) - water_filter_poll"),
+
+        ("waste_disposal_raw", "Waste_Disposal_Satisfaction (Value) - waste_disposal_satisfaction")
+        ("waste_disposal_run_id", "Waste_Disposal_Satisfaction (Run ID) - waste_disposal_satisfaction")
+        ("sent_on", "Waste_Disposal_Satisfaction (Time) - waste_disposal_satisfaction")
+
+        ("age_raw", "Age (Value) - demogs"),
+        ("age_time", "Age (Time) - demogs"),
+
+        ("gender_raw", "Gender_Test (Value) - demogs"),
+        ("gender_time", "Gender_Test (Time) - demogs"),
+
+        ("client_region_raw", "Response_4 (Value) - demogs")
```

Comment:
There should be a way of configuring Rapid Pro so that this is called Client_Region rather than Response_4

Reply:
Unfortunately once you have the flow running you cant edit it the API will continue picking the orignal value. I noticed it late tried to change it but it kept falling back to Response_4 in the json file

Reply:
Aaargh Rapid Pro.

This is the sort of problem which will hopefully always be picked up in the testing phase of a real project. Has Test-ReDSS "started" yet, or is it still in test? If the latter, you should probably make the change, otherwise we live with Response_4.

Reply:
Unfortunately I already have a couple of messages in the flows for the 'test pipeline'. I will keep it as Response_4 for now but will be careful in the next actual projects. Thanks alot @as2388